source/docs/articles/drupal/adding-yml-file.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Adding a services.yml File
+description: Learn how to add a services.yml file for Drupal 8.
+category:
+  - developing
+  - drupal
+keywords: drupal, services.yml, yml file
+---
+Drupal 8 includes the `sites/default/default.services.yml` configuration file to define services used, eliminating the need to hack core to swap functionality. To create and/or modify this file, you'll need to tweak some permissions, otherwise you will receive a **Permission denied** error.
+
+<div class="alert alert-danger">
+<h4>Warning</h4>
+It is not possible to modify these permissions with Git. Users working via Git should finish up their code changes and briefly switch to <a href="/docs/articles/sites/code/developing-directly-with-sftp-mode">SFTP Mode</a> to change the permissions.
```

Comment:
How/why is this different from settings.php? I thought I can add settings.php with git or sftp and not need to worry about permissions?